apps/dashboard/src/app/(dashboard)/interest-group/[id]/edit-card.tsx

```diff
@@ -33,11 +33,22 @@ export const InterestGroupEditCard: FC = () => {
     defaultValues: interestGroup,
   })
   return (
-    <div>
+    <Stack gap="xs">
       <FormComponent />
-      <Button variant="outline" color="red" style={{ marginTop: "1rem" }} onClick={open}>
-        <Icon icon="tabler:trash" />
-      </Button>
-    </div>
+      <div style={{ display: "flex", flexWrap: "wrap", alignItems: "center" }}>
+        <Button variant="outline" color="red" style={{ width: "55px", marginRight: "24px" }} onClick={open}>
+          <Icon icon="tabler:trash" />
+        </Button>
+        <Checkbox
+          label="Aktiv?"
+          onChange={() =>
+            edit.mutate({
+              ...interestGroup, values: { ...interestGroup, isActive: !interestGroup.isActive, },
+            })
+          }
+          checked={interestGroup.isActive}
+        />
```

Comment:
I am now a little unsure if this is needed. It may be more practical to add  a useCheckboxInput in the useInterestGroupWriteForm instead such that the update button activates the edit mutation

Reply:
Fixed